### ISSUE TITLE
R/L half keyboard shortcut moves window between displays 

### ISSUE THREAD

**Issue by igal1c0de4n** [rxhanson/RectanglePro-Community#152]
_New user here, maybe I've misunderstood expected behavior, but this looks like a bug:_

I use the default keyboard shortcuts for R/L halves:

"Left half" = command + option + LeftArrow 
"Right half" = command + option + RightArrow 

**The issue:**
Typing those shortcuts, after the window reaches it's intended location, the next typing of the shortcut moves the window to the next display. 

**Expected:**
Window should remain in the same display, perhaps moved to the other half of the screen (?)

![Screenshot 2022-10-29 at 10 28 12 PM](https://user-images.githubusercontent.com/1245766/198849417-3b541f47-07e3-4517-96d1-b9884019dc86.png)

**Comment by rxhanson** [comment]
You can adjust the "Repeated commands" setting in the Settings tab of the Preferences window to change this behavior.

**Comment by igal1c0de4n** [comment]
Perfect, found it 👍🏼  

Strangely, I did look for such an option. 
If to be more precise, I looked for the "Cycle 1/2 ... on half actions" option. It's implicitly available in the free version (and in Spectacle - which I was using until now). 

If purely associated with keyboard shortcuts (?) maybe having this option under that context would be easier to find.
But if this option affects a non-keyboard-shortcut, placing it only in the keyboard shortcut settings would be confusing.

Anyway, thanks for the tip. I consider the original issue resolved
